Aardwolf_Rich_Exits.lua

```diff
@@ -355,8 +355,25 @@ end
 --
 
 function alias_cexit(name, line, wildcards)
-    local index = tonumber(wildcards.index)
-    local cexit = cexits[index]
+    local index = wildcards.index
+    local cexit = nil
+    if index == "n" then
+        cexit = exits.north
+    elseif index == "e" then
+        cexit = exits.east
+    elseif index == "s" then
+        cexit = exits.south
+    elseif index == "w" then
+        cexit = exits.west
+    elseif index == "u" then
+        cexit = exits.up
+    elseif index == "d" then
+        cexit = exits.down
+    else
+        index = tonumber(wildcards.index)
+        cexit = cexits[index]
+    end
+
```

Comment:
Enable support for `cexit [neswud]` in addition to numbers. This is so I can bind the cexit command to my numpad directional keys and have it automatically handle door cexits if configured.